docs/glossary.md

```diff
@@ -24,7 +24,7 @@ permalink: /glossary
 - **Linkage error**: an abnormal condition of a classpath in which a
   "class has some dependency on another class; however, the
   latter class has incompatibly changed after the compilation of the
-  former class."<sup>[1](#myfootnote1)</sup> The reference can be
+  former class."<sup>[1](#myfootnote1)</sup> The dependency can be
   through a class literal, a field access, or a method invocation.
```

Comment:
```suggestion
  symbolic reference to a class literal, a field access, or a method invocation.
```

Reply:
That introduces a new term here that is not defined and not otherwise used on this page.

Reply:
@elharo I thought about the difference between "dependency" and "symbolic reference". The symbolic references in the constant pool of class files are the implementation detail of the "dependency" written in the [java.lang.LinkageError Javadoc](https://docs.oracle.com/javase/8/docs/api/java/lang/LinkageError.html).  Therefore the term "symbolic reference" does not belong to this doc.

Does this match your thought?

Reply:
It's simpler than that. We're quoting here from the official JDK docs, with citation. We use the same term they use. We do not need to introduce new and unfamiliar terms to define a linkage error here.

Reply:
Okay.